gadwords.php

```diff
@@ -19,140 +19,113 @@
+		$this->isPrestashop16 = version_compare(_PS_VERSION_, '1.7.0.0', '<');
 	}
 
+	/**
+	 * Install the module gadwords and manage the module Prestashop Ads
+	 *
+	 * @return bool
+	 */
 	public function install()
 	{
-		return parent::install() && $this->registerHook('backOfficeHeader');
+		$moduleManagement = new GAdwordsModuleManagement;
+
+		return parent::install() 
+			&& $moduleManagement->moduleManagement(self::PRESTASHOP_ADS_MODULE_NAME, self::PRESTASHOP_ADS_MODULE_ID);
 	}
 
-	public function hookBackOfficeHeader()
+	/**
+	 * Load Assets 
+	 *
+	 * @return void
+	 */
+	private function loadAssets()
 	{
-		if (strcmp(Tools::getValue('configure'), $this->name) === 0)
-		{
-			if (version_compare(_PS_VERSION_, '1.5', '>') == true)
-			{
-				$this->context->controller->addCSS($this->_path.'css/gadwords.css');
-				if (version_compare(_PS_VERSION_, '1.6', '<') == true)
-					$this->context->controller->addCSS($this->_path.'css/gadwords-nobootstrap.css');
-			}
-			else
-			{
-				echo '<link rel="stylesheet" href="'.$this->_path.'css/gadwords.css" type="text/css" />';
-				echo '<link rel="stylesheet" href="'.$this->_path.'css/gadwords-nobootstrap.css" type="text/css" />';
-			}
+		if ($this->isPrestashop16) {
+			$this->context->controller->addCSS('//fonts.googleapis.com/icon?family=Material+Icons');
 		}
+
+		$this->context->controller->addCSS($this->_path.'views/css/gadwords.css');
 	}
 
-	public function getContent()
+	/**
+	 * Get the module's configuration link for Prestashop 1.6 or Prestashop 1.7
+	 *
+	 * @return string
+	 */
+	private function getModulePrestashopAdsLink()
 	{
-		switch (Tools::strtolower($this->context->language->iso_code))
-		{
-			case 'be':
-				$landing_page = 'http://www.google.com/intl/fr/ads/get/prestashop75/index.html';
-				break;
-			case 'cz':
-				$landing_page = 'http://www.google.com/ads/get/prestashop1000/index.html';
-				break;
-			case 'de':
-				$landing_page = 'http://www.google.com/intl/de/ads/get/prestashop50/index.html';
-				break;
-			case 'fr':
-				$landing_page = 'http://www.google.com/intl/fr/ads/get/prestashop75/index.html';
-				break;
-			case 'gb':
-			case 'en':
-				$landing_page = 'http://www.google.co.uk/ads/get/prestashop75/index.html';
-				break;
-			case 'es':
-				$landing_page = 'http://www.google.com/intl/es/ads/get/prestashop75/index.html';
-				break;
-			case 'it':
-				$landing_page = 'http://www.google.com/intl/it/ads/get/prestashop75/index.html';
-				break;
-			case 'nl':
-				$landing_page = 'http://www.google.com/intl/nl/ads/get/prestashop75/index.html';
-				break;
-			case 'pl':
-				$landing_page = 'http://www.google.com/intl/pl/ads/get/prestashop250/index.html';
-				break;
-			case 'ro':
-				$landing_page = 'http://www.google.com/ads/get/prestashop200/index.html';
-				break;
-			default:
-				$landing_page = 'http://www.google.co.uk/adwords/start';
+		if (!Module::isInstalled(self::PRESTASHOP_ADS_MODULE_NAME)) {
+			return '';
```

Comment:
Instead of installing the other module at the same time as `gadword`, which I disapprove, why don't you create a link here which installs the other module?